zppy/pcmdi_diags.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
I recommend taking a look at https://github.com/E3SM-Project/zppy/blob/main/zppy/e3sm_diags.py or https://github.com/E3SM-Project/zppy/blob/main/zppy/global_time_series.py for examples of how to better modularize this code.